src/lib/rules/validate-set-cookie-header/validate-set-cookie-header.ts

```diff
@@ -0,0 +1,226 @@
+/**
```

Comment:
did you try this?
`const [nameValuePair, ...directivePairs]: Array<string> = setCookieValue.split(';');`